aten/src/ATen/native/native_functions.yaml

```diff
@@ -1646,6 +1646,12 @@
 - func: hamming_window.periodic_alpha_beta(int window_length, bool periodic, float alpha, float beta, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
   use_c10_dispatcher: full
 
+- func: kaiser_window(int window_length, float beta, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
```

Comment:
See note above about how we should derive arg order. I actually think PyTorch's ordering is a little silly, and agree it's more natural to put arguments like `beta` before arguments like `periodic`. However, we can fix that when we implement the NumPy names (that is, `kaiser` instead of `kaiser_window`). The one caveat with that plan is that NumPy's name for the Hann window is incorrect, but we'll get to that later (I filed an issue with NumPy for this https://github.com/numpy/numpy/issues/17303). 

This suggestion requires setting a default beta parameter. matlab uses 1/2, the NumPy documentation suggests 14, and TensorFlow's default is 12. 12 seems reasonable to me.